pysparkling/context.py

```diff
@@ -194,6 +193,9 @@ def parallelize(self, x, numSlices=None):
         :rtype: RDD
         """
         if numSlices is None or numSlices <= 1:
+            # pylint: disable=import-outside-toplevel, cyclic-import
```

Comment:
This was importable at the top level before. Can you elaborate why that cannot be done anymore?

Reply:
A sample of the error I'm getting:
```
pysparkling\__init__.py:8: in <module>
    from . import exceptions, fileio, streaming
pysparkling\streaming\__init__.py:2: in <module>
    from .context import StreamingContext
pysparkling\streaming\context.py:4: in <module>
    from .dstream import DStream
pysparkling\streaming\dstream.py:5: in <module>
    from ..rdd import EmptyRDD
pysparkling\rdd.py:23: in <module>
    from .context import Context
pysparkling\context.py:17: in <module>
    from .rdd import RDD
E   ImportError: cannot import name 'RDD'
```

The problem is that everything is depending on RDD/Row and Context. I tried playing around a bit with the order of the imports, but I failed to find a good fit.

And both rdd & context have the other imported...
As RDD is only needed 2x in context.py, I opted to make it inline there. (Context appears 165x in rdd.py).

Reply:
This was working before. This is a blocker for me. See below in RDD file for a suggestion.